### PR TITLE
Add pphuman doc

### PR DESCRIPTION
**Pull request by jerrywgz**
1. add pphuman doc, demo video will update in next pr
2. add enable_attr, enable_action as input arg

### CODE REVIEW

File: deploy/pphuman/README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  *  [2. 多目标跟踪](##多目标跟踪)
+  *  [3. 跨镜跟踪](##跨镜跟踪)
+  *  [4. 属性识别](##属性识别)
+  *  [5. 行为识别](##行为识别)
+
+
+
+## 环境准备
+
+环境要求： PaddleDetection版本 > release/2.4
```
Comment (heavengate) [inline deploy/pphuman/README.md]: ">="?

File: deploy/pphuman/README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+| 参数 | 是否必须|含义 |
+|-------|-------|----------|
+| --config | Yes | 配置文件路径 |
+| --model_dir | Option | PP-Human中各任务模型路径，优先级高于配置文件 |
+| --image_file | Option | 需要预测的图片 |
+| --image_dir  | Option |  要预测的图片文件夹路径   |
+| --video_file | Option | 需要预测的视频 |
+| --camera_id | Option | 用来预测的摄像头ID，默认为-1(表示不使用摄像头预测，可设置为：0 - (摄像头数目-1) )，预测过程中在可视化界面按`q`退出输出预测结果到：output/output.mp4|
+| --enable_attr| Option | 是否进行属性识别 |
+| --enable_attr| Option | 是否进行行为识别 |
```
Comment (heavengate) [inline deploy/pphuman/README.md]: enable_action

File: deploy/pphuman/README.md (hunk 1 above)
Comment (jerrywgz) [reply deploy/pphuman/README.md]: done, thx

File: deploy/pphuman/README.md (hunk 2 above)
Comment (jerrywgz) [reply deploy/pphuman/README.md]: done ,thx

**Review by YixinKristy — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**